### ISSUE TITLE
Acme.exe v0.14 for Windows Doesn't Start

### ISSUE THREAD

**Issue by trevor-ian-nguyen** [caerwynj/acme-sac#38]
I downloaded Acme.exe v0.14 to a Windows 10 Enterprise 64-bit system. When I run the executable, the app never starts (its window never displays, and it does not appear in the Task Manager list)

I tried to run the app as the normal logged in user and also as administrator.

**Comment by bpotvin** [comment]
I had a similar problem last night. I pulled the repo here, but when I tried to run the Acme.exe tossed an error saying that it was an un-supported 16-bit application. Looking at the binary, though, it's for sure a PE executable, albeit one that's apparently corrupt. So, I fixed things by using an older acme windows build. The google code archive site for acme-sac has a number of older builds available in the "downloads" section:

https://code.google.com/archive/p/acme-sac/

I used the Acme-0.13.zip archive. After making sure it ran, I cloned the repo here, replaced the fuxor'd acme.exe with the one from the Acme-0.13.zip archive and then went about rebuilding the /sys/ directory using VS2017. That (eventually) worked and now everything's good.

**Comment by eekee** [comment]
Same here, although it doesn't tell me "16-bit".  Thanks for the instructions.

**Comment by aks2161989** [comment]
@bpotvin Thank you for the instructions! I was able to build `iacme.exe` using your instructions. However, there is a problem:` iacme.exe` does not launch when I double-click it. I am on Windows 10. Were you able to get it working?

**Comment by aks2161989** [comment]
@bpotvin I moved iacme.exe  from the `\acme-sac-master\sys\emu\Nt` directory to `\acme-sac-master`. I can now open iacme.exe by double clicking it, but the fonts are all messed up

![Screenshot (32)](https://user-images.githubusercontent.com/20129646/77463686-05043680-6e2c-11ea-9d4c-f3e43cd9e6a7.png)

Do the fonts render correctly for you?

**Comment by bpotvin** [comment]
@aks2161989,

Yes, everything works for me. I haven't seen anything like that before; no idea what it might be.

Regarding fonts, I'm using only the fonts that are included here. I know there are other fonts available, for example from the vitanuova site, but I've not tried using any others.

On startup, (see /sys/emu/port/main.c), the code looks for "INFERNO" and "ROOT" environment vars to define the root directory. If neither are set, the default appears to be the directory from which acme.exe is loaded. I use $ROOT for other things and that created a problem once; that might be something to check.

Also, I'm doing 32-bit builds; I've not tried to do a 64-bit build and have no idea if it would work.

I did a complete re-build from scratch today. I'm attaching two files, one with the steps to do the rebuild and the second with the window output of the build from within acme.

[README_REBUILD.TXT](https://github.com/caerwynj/acme-sac/files/4379183/README_REBUILD.TXT)

[build-output.txt](https://github.com/caerwynj/acme-sac/files/4379184/build-output.txt)

**Comment by aks2161989** [comment]
@bpotvin Thank you for taking the time to rebuild acme and also for saving the build log. I really appreciate it. Unfortunately, I tried following the exact steps in your build-log.txt and the build is successful (as before). But I get the same distorted display/fonts as my earlier comment. I guess I will have to dig deep into the source code/config files to determine the problem. Thank you once again!

**Comment by vic-codes** [comment]
The distortion only happens to me on Windows 7 if I have my taskbar fixed on the sides.

**Comment by eekee** [comment]
I have my taskbar fixed to the side in Windows 10.

On Mon, Nov 15, 2021, at 1:54 AM, V. Fabiano wrote:
> The distortion only happens to me on Windows 7 if I have my taskbar 
> fixed on the sides.
>
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/caerwynj/acme-sac/issues/38#issuecomment-968440875

-- 
Formerly known as Ethan Gardener

**Comment by meetkool** [comment]
@bpotvin do u still have the build ?  can u release it
